synapse/models/economic.py

```diff
@@ -61,10 +67,29 @@ def getModelDefs(self):
                 ('econ:fin:tick', ('guid', {}), {
                     'doc': 'A sample of the price of a security at a single moment in time'}),
 
-                # econ:acct:bill
-                # econ:bank:us:aba:rtn ( ABA Routing Number )
-                # econ:bank:us:account = (econ:bank:us:aba:rtn, acct)
-                # econ:bank:swift:...
+                ('econ:bank:account:type:taxonomy', ('taxonomy', {}), {
+                    'doc': 'A bank account type taxonomy.'}),
+
+                ('econ:bank:account', ('guid', {}), {
+                    'doc': 'A bank account.'}),
+
+                ('econ:bank:balance', ('guid', {}), {
```

Comment:
should we consider econ:bank:acct:balance? The balance is linked to an acct not a bank

Reply:
`econ:bank:account:balance` is already the secondary property on the `econ:bank:account` node.